src/Gui/DlgUnitsCalculator.ui

```diff
@@ -34,13 +37,19 @@
       </widget>
      </item>
      <item>
-      <widget class="Gui::InputField" name="UnitInput">
+      <widget class="QLineEdit" name="UnitInput">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
+       <property name="toolTip">
+        <string>specify here the result unit</string>
```

Comment:
Grammar here need tweaking. Perhaps: `Specify the result unit in this field` or `In this field, specify the result unit` ?